site/app/libraries/database/DatabaseUtils.php

```diff
@@ -4,9 +4,18 @@
 
 namespace app\libraries\database;
 
+use app\libraries\DateUtils;
+
 class DatabaseUtils {
     public static function formatQuery($sql, $params): string {
+        if (is_null($params)) {
+            return '';
+        }
```

Comment:
So given the above, let's make this change:

```suggestion
    public static function formatQuery(string $sql, ?array $params): string {
        if ($params === null) {
            return $sql;
        }
```

I forgot to add types in my original PR, so let's add them here, where this should be the expected shape given the bit about null. Also using `=== null` is a micro-optimization over `is_null`, but worth preferring as it avoids the overhead of a function call to do the exact same thing as equality. Finally, given null parameters, we should still have a string for the query to run, so should just return `$sql` as is.